### ISSUE TITLE
Misleading error message on ARM

### ISSUE THREAD

**Issue by eddywestbrook** [GaloisInc/saw-script#1803]
If the `llvm_verify_x86` command is called on an ARM file, it gives the error message “Unsupported x86 feature: 32-bit ELF format”, rather than saying something more helpful.

**Comment by RyanGlScott** [comment]
This should be relatively simple to check. This is the code that produces the `Unsupported x86 feature: 32-bit ELF format` error:

https://github.com/GaloisInc/saw-script/blob/c5c4051d58e88a5d29509d5f1df3b167649ee294/src/SAWScript/X86.hs#L274-L277

In addition to matching on the `headerClass`, we can also match on the `headerMachine` to see if it is `EM_X86_64`. If not, we could print out the `headerMachine` to give the users a clue about what went wrong. (In the case of ARM32, that would be `EM_ARM`.)

**Comment by eddywestbrook** [comment]
That certainly makes sense. It does, however, also look like `llvm_verify_x86` doesn't support verifying 32-bit code anyway, though...?

**Comment by RyanGlScott** [comment]
Correct, `llvm_verify_x86` only supports x86-64, a 64-bit architecture.
